Tribler/community/doubleentry/community.py

```diff
@@ -0,0 +1,325 @@
+        super(DoubleEntryCommunity, self).initialize()
+
+    @classmethod
+    def get_master_members(cls, dispersy):
+        # generated: Wed Dec  3 10:31:16 2014
+        # curve: NID_sect571r1
+        # len: 571 bits ~ 144 bytes signature
+        # pub: 170  3081a7301006072a8648ce3d020106052b810400270381920004059f45b75d63f865e3c7b350bd3ccdc99dbfbf76f
+        #           dfb524939f070223c3ea9ea5d0536721cf9afbbec5693798e289b964fefc930961dfe1a7f71c445031434aba637bb9
+        #           3b947fb81603f649d4a08e5698e677059b9d3a441986c16f8da94d4aa2afbf10fe056cd65741108fe6a880606869c
+        #           a81fdcb2db302ac15905d6e75f96b39ccdaf068bdbbda81a6356f53f7ce4e
+        # pub-sha1 f66a50b35c4a0d45abd0052f574c5ecc233b8e54
+        # -----BEGIN PUBLIC KEY-----
+        # MIGnMBAGByqGSM49AgEGBSuBBAAnA4GSAAQFn0W3XWP4ZePHs1C9PM3Jnb+/dv37
+        # Ukk58HAiM+qepdBTZyHPmvu+xWk3mOKJuWT+/JMJYd/hp/ccRFAxQ0q6Y3u5O5R/
+        # uBYD9knUoI5WmOZ3BZudOkQZhsFvjalNSqKvvxD+BWzWV0EQj+aogGBoacqB/cst
+        # swKsFZBdbnX5aznM2vBovbvagaY1b1P3zk4=
+        # -----END PUBLIC KEY-----
+        master_key = "3081a7301006072a8648ce3d020106052b810400270381920004059f45b75d63f865e3c7b350bd3ccdc99dbfbf76f" + \
+                     "dfb524939f0702233ea9ea5d0536721cf9afbbec5693798e289b964fefc930961dfe1a7f71c445031434aba637bb9" + \
+                     "3b947fb81603f649d4a08e5698e677059b9d3a441986c16f8da94d4aa2afbf10fe056cd65741108fe6a880606869c" + \
+                     "a81fdcb2db302ac15905d6e75f96b39ccdaf068bdbbda81a6356f53f7ce4e"
+        master_key_hex = master_key.decode("HEX")
+        master = dispersy.get_member(public_key=master_key_hex)
+        return [master]
+
+    def initiate_meta_messages(self):
+        return super(DoubleEntryCommunity, self).initiate_meta_messages() + [
+            Message(self, SIGNATURE_REQUEST,
+                    MemberAuthentication(),
+                    PublicResolution(),
+                    DirectDistribution(),
+                    CandidateDestination(),
+                    SignatureRequestPayload(),
+                    self._check_signature_request,
+                    self._on_signature_request),
+            Message(self, SIGNATURE_RESPONSE,
+                    MemberAuthentication(),
+                    PublicResolution(),
+                    DirectDistribution(),
+                    CandidateDestination(),
+                    SignatureResponsePayload(),
+                    self._check_signature_response,
+                    self._on_signature_response)]
+
+    def initiate_conversions(self):
+        return [DefaultConversion(self), DoubleEntryConversion(self)]
+
+    def publish_signature_request_message(self, candidate):
+        """
+        Creates and sends out signature_request message.
+        """
+        self._logger.info("Sending signature request.")
+        message = self.create_signature_request_message(candidate)
+        self.dispersy.store_update_forward([message], True, True, True)
```

Comment:
The _forward method is private and gives a pylint error. There are two solutions: 
- Use store_update_forward([message], false, false, true)
- Change the method to public. It is accessed publicly in a lot of other places aswell, so this might be a good change overall. 

What do you prefer?